docs/latest/concepts/deployment.md

```diff
@@ -8,8 +8,8 @@ deployed to any system or platform that can run a Deno based web server.
 
 Here are instructions for specific providers / systems:
 
-- [Deno Deploy](#deno-deploy)
-- [Docker](#docker)
+- [Deno Deploy](#-deno-deploy)
+- [Docker](#-docker)
```

Comment:
Dunno if this is supposed to be like this, but the anchors did not work for me before.

Reply:
mh yeah that looks odd. We should adapt our code to get rid of the leading dash. Can happen in another PR.